canonical/ubuntu.com
# WD-1466 add notice for posts written/updated over 1 year ago

## Done

- Added blog date notice for posts written/updated over a year ago.

## QA

- Check out the demo here and compare the following article to the screenshot below and in the task:
  - [LXD 5.0 LTS is now available](https://ubuntu-com-12833.demos.haus/blog/lxd-5-0-lts-is-now-available) 
    - this post is just over a year old and should say: "This article is more than **1 year old.**"
  - [Tutorials for everyone from everyone](https://ubuntu-com-12833.demos.haus/blog/tutorials-for-everyone-from-everyone) 
    - this post is just over 3 years old but was also updated about three years ago and should say: "This article was last updated **3 years ago.**"
  - [Top 6 projects from our Hackathon](https://ubuntu-com-12833.demos.haus/blog/top-6-projects-from-our-hackathon) 
    - this post is over a year old but was updated within the last year and should not have the blog date notice.
  - [Charmed Kubeflow is now available on AWS Marketplace](https://ubuntu-com-12833.demos.haus/blog/kubeflow-appliance-aws) 
    - this post is less than a year old and should not have the blog date notice.

## Issue / Card

- [WD-1466](https://warthogs.atlassian.net/browse/WD-1466)

## Screenshots

![Screenshot 2023-03-17 at 10 15 16](https://user-images.githubusercontent.com/48949356/235230935-821f0725-78a2-40c6-a4ad-b3c0f5ce3c33.png)



## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-1466]: https://warthogs.atlassian.net/browse/WD-1466?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
